projects/ngx-prx-styleguide/tsconfig.lib.prod.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
angular 9 libraries should not use Ivy https://indepth.dev/the-angular-ivy-guide-for-library-authors/